rotors_gazebo_plugins/src/gazebo_mavlink_interface.cpp

```diff
@@ -18,226 +19,1019 @@
  * limitations under the License.
  */
 
+
 #include "rotors_gazebo_plugins/gazebo_mavlink_interface.h"
+#include "rotors_gazebo_plugins/geo_mag_declination.h"
 
 namespace gazebo {
 
-GazeboMavlinkInterface::GazeboMavlinkInterface()
-    : ModelPlugin(),
-      node_handle_(0) {}
+// Set global reference point
+// Zurich Irchel Park: 47.397742, 8.545594, 488m
+// Seattle downtown (15 deg declination): 47.592182, -122.316031, 86m
+// Moscow downtown: 55.753395, 37.625427, 155m
+
+// Zurich Irchel Park
+static const double lat_zurich = 47.397742 * M_PI / 180;  // rad
```

Comment:
Fixed style. Haven't moved to header, not sure whether this is the best thing to do?